### DISCUSSION TITLE
GDExtension 1.0.0 Branch?

### DISCUSSION THREAD

**Discussion by h-sigma** [OctoD/godot-gameplay-systems#55] [General]
Would it be possible to add a branch for the 1.0.0 version? Mainly as a way to entice contributors.

**Comment by OctoD** [comment]
Will do, that branch actually will be just a collection of binaries from the extensions composing this addon (since godot asset store cannot let you redistribuite bundles). 

Also, this repo will become the "front" for all issues related to each separated system.
